architecture-building-systems/CityEnergyAnalyst
# 3163 new objective functions for dc

As described in #3163 this branch introduces 'heat release' and 'system energy demand' as new objective functions of the centralised optimisations (i.e. Supply System Part II) with **district cooling** networks. 

To test:

1. Choose any CEA-scenario
2. Run everything up to the centralised optimisation script
3. Select a set of objective functions you would like run the optimisation with:
![Screen Shot 2022-07-20 at 11 23 04 AM](https://user-images.githubusercontent.com/93093343/179889639-1366ab01-7303-48ed-9e29-5934f2a7e2d0.png)

4. Run the script.
5. Check 'scenario/outputs/data/calibration/clustering/checkpoints/ind_%(ind_num)s_buildings_building_scale_performance.csv' to see evolution of objective functions throughout the different generations of the genetic algorithm.
